.gitattributes

```diff
@@ -5,10 +5,13 @@
 /.gitignore export-ignore
 /.scrutinizer.yml export-ignore
 /.travis.yml export-ignore
+/build-phar.sh export-ignore
 /CHANGELOG.md export-ignore
 /RATIONALE.md export-ignore
 /README.md export-ignore
 /SECURITY.md export-ignore
 /phpunit.sh export-ignore
 /phpunit.xml.dist export-ignore
-/other/ide_stubs export-ignore
+/psalm.xml export-ignore
+psalm-autoload.php export-ignore
```

Comment:
why `psalm-autoload.php` is not with absolute path?

Reply:
Mere oversight. Thanks for catching it. I'll update the PR.